File: Data Structures & Algorithms/concatenation-of-array/submission-0.py
class Solution:
    def getConcatenation(self, nums: List[int]) -> List[int]:
        n = len(nums)
        size = n * 2
        res = [0] * size

        for i in range(n):
            res[i], res[i+n] = nums[i], nums[i]
        
        return res

        

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.java
class Solution {
    public int longestConsecutive(int[] nums) {
        int res=0;
        Set<Integer> numSet = new HashSet<>();
        for(int n : nums){
           numSet.add(n);
        }
        for(int n : nums){
            if(!numSet.contains(n-1)){
                int len =1;
                while(numSet.contains(n+len)){
                    len++;
                }
                res = Math.max(len,res);
            }     
        }
        return res;
    }
    
}
